File: prime5to50.py
#!/usr/bin/python

def is_prime(n):
    p = True;
    for index in range(2, n):
        if(n % index == 0):
            p = False;
    return p

for index in range(5,50):
    if(is_prime(index)):
        print("%d is a prime number" %index)

File: prime5to50.ts
function prime5to50(): string {
	let message : string = ""

	let tmpmsg = " is a prime number&#13;&#10;";
	for(var i = 5; i <= 50; i ++){
		if(isPrime(i)){
			message = message + i + tmpmsg;
		}
	}

	// TODO: fill up the message string with the proper content.

	return message
}

function isPrime(n): boolean {
	var p = true;
	for (var i=2; i < n; i++){
		if(n % i == 0){
			p = false;
		}
	}
	return p;
}

// You may use additional helper functions.

document.body.innerHTML = prime5to50()
